src/game.cpp

```diff
@@ -9143,7 +9108,9 @@ void game::draw_item_filter_rules(WINDOW *window, int rows)
 std::string game::ask_item_priority_high(WINDOW *window, int rows)
 {
     for (int i = 0; i < rows - 1; i++) {
-        mvwprintz(window, i, 1, c_black, "%s", "                                                        ");
+        for (int j = 1; j < getmaxx(window) - 1; j++) {
```

Comment:
You can use one of the many `std::string` constructors here:
```C++
mvwprintz(window, i, 1, c_black, std::string(getmaxx(window) - 2, ' ').c_str());
```
That constructor is `string(size_t n, char c)`, it fills the string with `n` times the character `c`.

And this could even be put into another function with a meaningful signature like `clear_line(WINDOW*, int line, int length)`, which also explains *why* it's done (to clear the line from any previous content, not because we want to show a bunch of space characters).